modules/apps/object/object-test/src/testFunctional/tests/ObjectActions.testcase

```diff
@@ -139,6 +139,41 @@ definition {
 			actionName = "Action Name");
 	}
 
+	@description = "LPS-145665. Verify that you can edit the Action name"
+	@priority = "4"
+	test CanEditActionName {
+		ObjectAdmin.addObjectViaAPI(
+			labelName = "Custom Object 145",
```

Comment:
It should be `LPS-145665 - Verify` following the convention in other tests

Reply:
Fixed